ghwatch.py

```diff
@@ -330,8 +334,8 @@ def main():
         exit(1)
     auto_update = config.get('auto_update', 0)
     ghmeta = GhMeta(config['meta'])
-    g = Github(config['ghtoken'])
-    u = g.get_user()
+    gh = Github(config['ghtoken'])
+    user = gh.get_user()
```

Comment:
one possible mini-simplification if you retouch 
```diff
-    gh = Github(config['ghtoken'])
-    user = gh.get_user()
+    user = Github(config['ghtoken']).get_user()
```

Reply:
I slightly prefer keeping the parent instance `gh` assigned to a separate variable instead of chaining these. It's only an extra local doesn't really get in the way of anything :slightly_smiling_face: